src/schema/v2/admin/featureFlags.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLFieldConfig,
+  GraphQLID,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { merge, sortBy } from "lodash"
+import { ResolverContext } from "types/graphql"
+import { date } from "../fields/date"
+import { base64 } from "lib/base64"
+
+export const FeatureFlagType = new GraphQLObjectType<any, ResolverContext>({
+  name: "FeatureFlag",
+  fields: {
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+      resolve: (data) => {
+        return base64(data.name)
+      },
+    },
+    description: {
+      type: GraphQLString,
+    },
+    environments: {
+      type: new GraphQLList(
+        new GraphQLObjectType({
+          name: "FeatureFlagEnvironments",
+          fields: {
+            name: {
+              type: GraphQLString,
+            },
+            displayName: {
+              type: GraphQLString,
+            },
+            enabled: {
+              type: GraphQLBoolean,
```

Comment:
Lists + their members should always be non-nullable.
Booleans should always be non-nullable.
And I'd check to see if name and displayName can be too?